docs/source/index.rst

```diff
@@ -1,50 +1,42 @@
 .. Composer documentation master file
 
-Composer (Beta)
-===============
+Composer
+========
 
-MosaicML ``Composer`` contains a library of ML training efficiency methods, and a modular approach to compose them
-together to train deep neural networks. We aim to ease the transition from research to industry through reproducible
-code and rigorous benchmarking. With Composer, speed-up or accuracy-boosting methods can be easily composed into
-complete recipes.
+Composer provides well-engineered implementations of efficient training methods
+to give the tools that help you train a better model for cheaper.
 
-The library features:
+Using Composer, you can:
 
-* Implementation of 20+ efficiency methods curated from the research community
-* Standardized approach to implement and compose efficiency methods, extended from two-way callbacks
-  (`Howard et al, 2020 <https://arxiv.org/abs/2002.04688>`_)
-* Easy way to access our methods either directly for your trainer loops, or through the Composer trainer
+- Train an ImageNet model to 76.1% accuracy for $37 (`with vanilla PyTorch`:$127)
+- Train a GPT-2 125M to a perplexity of 23.9 for $148 (`with vanilla PyTorch`: $255)
+- Use start-of-the-art implementations of methods to speed up your own training loop.
 
-.. note::
-    MosaicML Composer is currently in **beta**, so the API is subject to change.
+At MosaicML, we are focused on making training ML models accessible. To do this,
+we continually productionize state-of-the-art academic research on efficient model
+training, and also study the `combinations`` of these methods in order to ensure
+that model training is ✨ as efficient as possible ✨.
```

Comment:
[nit] Perhaps we can add an "About us" section below by the community bits and keep the first part of this document Composer specific.  Document flow would then be
1. Composer TL;DR
2. Real results and what you can do with Composer
3. Highlight key Composer features
4. Getting Started
5. What MosaicML is about
6. Community plug